worlds/kh2/Rules.py

```diff
@@ -83,6 +83,8 @@ def hundred_acre_unlocked(self, state: CollectionState, amount) -> bool:
         return state.has(ItemName.TornPages, self.player, amount)
 
     def level_locking_unlock(self, state: CollectionState, amount):
+        if self.world.options.Promise_Charm:
```

Comment:
I feel like it not checking if the player actually has the Promise Charm could create issues, such as having to grind out to Level 50 without any world unlocks.

Reply:
oh you right